public/usage-examples/graphics/draw_line_on_window/draw_line_on_window-1-simple-oop.cs

```diff
@@ -0,0 +1,30 @@
+using SplashKitSDK;
+
+namespace draw_line_on_window
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create Window
+            Window start = new Window("draw_line_on_window", 600, 600);

```

Comment:
I would suggest changing the name of this window to be something different. On the website preview this line comes up as the "function" highlighted. because it has the same syntax. A simple fix would be to just remove the underscore. Make sure you change this over all of the program files so they remain the same.

Example of what im meaning - 
![image](https://github.com/user-attachments/assets/215040fb-74d4-4c7f-89c1-fc744d0034bf)


Otherwise, ive reviewed all the code files and they run correctly. I'm happy to approve this once that change is done.

Reply:
This change has been implemented